cmd/kubeadm/app/util/logfile.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+// kubeadm record file path and name define
+const (
+	kubeadmLogFolder   = "/var/lib/kubeadm"
+	kubeadmLogFile     = "kubeadm.log"
+	kubeadmLogTestFile = "kubeadm-log-file-test"
+)
+
+var kubeadmLogTestFolder = os.TempDir()
```

Comment:
@chenyb4 so this solves the test issues.

potentially we can move both `kubeadmLogTestFile` and `kubeadmLogTestFolder` to the body of the test.
but this isn't a big problem.